src/main/java/org/spongepowered/common/mixin/core/server/PlayerListMixin.java

```diff
@@ -368,6 +369,9 @@ public EntityPlayerMP recreatePlayerEntity(final EntityPlayerMP playerIn, int ta
         ((org.spongepowered.api.world.World) worldServer).spawnEntity(spongeEntity);
         this.playerEntityList.add(newPlayer);
         newPlayer.connection.sendPacket(new SPacketPlayerListItem(SPacketPlayerListItem.Action.UPDATE_GAME_MODE, newPlayer));
+        for (DataManipulator<?, ?> container : ((Player) playerIn).getContainers()) {
```

Comment:
Should be using the custom containers instead of all containers.

Reply:
Wouldn't that be missing vanilla manipulators ?